src/api/kube.ts

```diff
@@ -1096,6 +1111,35 @@ export class KubeHelper {
     }
   }
 
+  async isCRDEqual(crdClusterName: string, crdFilePath: string): Promise<boolean> {
+    const { spec: crdFileSpec } = this.safeLoadFromYamlFile(crdFilePath) as V1beta1CustomResourceDefinition
+    const { spec: crdClusterSpec } = await this.getCrd(crdClusterName)
+    const { validation: { openAPIV3Schema : { properties: crdFileProps = '' } = {} } = {} } = crdFileSpec
+    const { validation: { openAPIV3Schema : { properties: crdClusterProps = '' } = {} } = {} } = crdClusterSpec
+
+    if (crdFileSpec.versions && crdClusterSpec.versions) {
+      const [{ name: crdFileVersion }] = crdFileSpec.versions.filter(_ => {
+        return _.storage === true
+      })
+
+      const [{ name: crdClusterVersion }] = crdClusterSpec.versions.filter(_ => {
+        return _.storage === true
+      })
+
+      if (crdFileVersion !== crdClusterVersion) {
```

Comment:
I am not sure but if we compare object then they are always different. 
Let's compare just names.
`crdFileVersion.name !== crdClusterVersion.name`